index.html

```diff
@@ -71,8 +70,210 @@ <h2>I'am a full-stack Web developer</h2>
+                        <div class="card-title">
+                            Aura Patient Record System
+                        </div>
+                        <div class="card-discription">
+                            A record system for patients info and treatment data at AURA health care clinic.
+                        </div>
+                        <div class="card-lang">
+                            <div>html</div>
+                            <div>css</div>
+                            <div>php</div>
+                            <div>laravel</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="seemore">
+                    <a href="">See Project</a>
+                </div>
+            </div>
+            <div class="card-block">
+                <div class="card-pic">
+                    <div class="card-info">
+                        <div class="card-title">
+                            Aura Patient Record System
+                        </div>
+                        <div class="card-discription">
+                            A record system for patients info and treatment data at AURA health care clinic.
+                        </div>
+                        <div class="card-lang">
+                            <div>html</div>
+                            <div>css</div>
+                            <div>php</div>
+                            <div>laravel</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="seemore">
+                    <a href="">See Project</a>
+                </div>
+            </div>
+            <div class="card-block">
+                <div class="card-pic">
+                    <div class="card-info">
+                        <div class="card-title">
+                            Aura Patient Record System
+                        </div>
+                        <div class="card-discription">
+                            A record system for patients info and treatment data at AURA health care clinic.
+                        </div>
+                        <div class="card-lang">
+                            <div>html</div>
+                            <div>css</div>
+                            <div>php</div>
+                            <div>laravel</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="seemore">
+                    <a href="">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!--About Me-->
+    <section class="about-me">
+        <div class="home-content">
+            <div class="heading">
+                About me
+            </div>
+            <div class="bio">
+                I am full-stack web developer. Allow me to make your business connect with many people. Hope we can work together.
+            </div>
```

Comment:
- [ ] You are so far close to finishing the mobile version 🚶‍♂️ . However, we need to make sure that we use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) tags. Therefore, I suggest that for the paragraphs, we use `p` tags to improve the portfolio accessibility 👍 .